<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>powersource_window</class>
 <widget class="QMainWindow" name="powersource_window">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>610</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>460</width>
    <height>610</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>460</width>
    <height>610</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PSLab Power Source</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_pv1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>201</width>
      <height>261</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_pv1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string>Voltage Source 1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinner_pv1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::PlusMinus</enum>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="suffix">
      <string> V</string>
     </property>
     <property name="minimum">
      <double>-5.000000000000000</double>
     </property>
     <property name="maximum">
      <double>5.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
    <widget class="QSlider" name="slider_pv1">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>20</width>
       <height>211</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="sliderPosition">
      <number>500</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBothSides</enum>
     </property>
     <property name="tickInterval">
      <number>14</number>
     </property>
    </widget>
    <widget class="QDial" name="dial_pv1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>151</width>
       <height>151</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>500</number>
     </property>
     <property name="notchTarget">
      <double>10.000000000000000</double>
     </property>
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_pv2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>201</width>
      <height>261</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_pv2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string>Voltage Source 2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinner_pv2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::PlusMinus</enum>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="suffix">
      <string> V</string>
     </property>
     <property name="minimum">
      <double>-5.000000000000000</double>
     </property>
     <property name="maximum">
      <double>5.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
    <widget class="QSlider" name="slider_pv2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>20</width>
       <height>211</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>660</number>
     </property>
     <property name="value">
      <number>330</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBothSides</enum>
     </property>
     <property name="tickInterval">
      <number>14</number>
     </property>
    </widget>
    <widget class="QDial" name="dial_pv2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>151</width>
       <height>151</height>
      </rect>
     </property>
     <property name="maximum">
      <number>660</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>330</number>
     </property>
     <property name="notchTarget">
      <double>10.000000000000000</double>
     </property>
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_pcs">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>300</y>
      <width>201</width>
      <height>261</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_pcs">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string>Current Source</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinner_pcs">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::PlusMinus</enum>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="suffix">
      <string> mA</string>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>30.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QSlider" name="slider_pcs">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>20</width>
       <height>211</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>30</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBothSides</enum>
     </property>
     <property name="tickInterval">
      <number>1</number>
     </property>
    </widget>
    <widget class="QDial" name="dial_pcs">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>151</width>
       <height>151</height>
      </rect>
     </property>
     <property name="maximum">
      <number>30</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="notchTarget">
      <double>-0.500000000000000</double>
     </property>
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_pv3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>201</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_pv3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string>Voltage Source 3</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinner_pv3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::PlusMinus</enum>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="suffix">
      <string> V</string>
     </property>
     <property name="minimum">
      <double>-5.000000000000000</double>
     </property>
     <property name="maximum">
      <double>5.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
    <widget class="QSlider" name="slider_pv3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>20</width>
       <height>211</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>330</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBothSides</enum>
     </property>
     <property name="tickInterval">
      <number>14</number>
     </property>
    </widget>
    <widget class="QDial" name="dial_pv3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>151</width>
       <height>151</height>
      </rect>
     </property>
     <property name="maximum">
      <number>330</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="notchTarget">
      <double>4.000000000000000</double>
     </property>
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dial_pv1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>powersource_window</receiver>
   <slot>power_dial_changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>453</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_pv2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>powersource_window</receiver>
   <slot>power_dial_changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>454</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_pcs</sender>
   <signal>valueChanged(int)</signal>
   <receiver>powersource_window</receiver>
   <slot>power_dial_changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>389</y>
    </hint>
    <hint type="destinationlabel">
     <x>453</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_pv3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>powersource_window</receiver>
   <slot>power_dial_changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_pv1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>powersource_window</receiver>
   <slot>power_slider_changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>451</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_pv2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>powersource_window</receiver>
   <slot>power_slider_changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_pcs</sender>
   <signal>valueChanged(int)</signal>
   <receiver>powersource_window</receiver>
   <slot>power_slider_changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>420</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>448</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_pv3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>powersource_window</receiver>
   <slot>power_slider_changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>452</x>
     <y>411</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinner_pv1</sender>
   <signal>valueChanged(double)</signal>
   <receiver>powersource_window</receiver>
   <slot>power_spinner_changed(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>451</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinner_pv2</sender>
   <signal>valueChanged(double)</signal>
   <receiver>powersource_window</receiver>
   <slot>power_spinner_changed(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinner_pcs</sender>
   <signal>valueChanged(double)</signal>
   <receiver>powersource_window</receiver>
   <slot>power_spinner_changed(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>513</y>
    </hint>
    <hint type="destinationlabel">
     <x>451</x>
     <y>535</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinner_pv3</sender>
   <signal>valueChanged(double)</signal>
   <receiver>powersource_window</receiver>
   <slot>power_spinner_changed(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>522</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>468</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinner_pv1</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>powersource_window</receiver>
   <slot>power_spinner_changed(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>450</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinner_pv2</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>powersource_window</receiver>
   <slot>power_spinner_changed(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>448</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinner_pcs</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>powersource_window</receiver>
   <slot>power_spinner_changed(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>513</y>
    </hint>
    <hint type="destinationlabel">
     <x>450</x>
     <y>505</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinner_pv3</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>powersource_window</receiver>
   <slot>power_spinner_changed(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>530</y>
    </hint>
    <hint type="destinationlabel">
     <x>8</x>
     <y>432</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>power_dial_changed(int)</slot>
  <slot>power_slider_changed(int)</slot>
  <slot>power_spinner_changed(double)</slot>
  <slot>power_spinner_changed(QString)</slot>
 </slots>
</ui>
